collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-1.md

```diff
@@ -0,0 +1,129 @@
+
+- Windows builds no longer get stuck maximized under certain conditions ([GH-84504](https://github.com/godotengine/godot/pull/84504)).
+
+- Areas with a gravity influence now correctly forget about bodies leaving their range ([GH-82961](https://github.com/godotengine/godot/pull/82961)).
+
+- Baking lighting information for VoxelGI now correctly handles MultiMesh and CSG instances ([GH-81616](https://github.com/godotengine/godot/pull/81616)).
+
+- OpenXR is now included in macOS builds ([GH-79614](https://github.com/godotengine/godot/pull/79614)).
+
+- The latest version of MoltenVK SDK (1.3.275.0) is now supported for the macOS platform builds ([GH-87305](https://github.com/godotengine/godot/pull/87305)).
+
+- In the previous release we have enabled exceptions for Android, iOS, and Web builds; they are now correctly re-disabled in the buildsystem ([GH-84328](https://github.com/godotengine/godot/pull/84328)).
+
+This release is built from commit [`b9008f3d5`](https://github.com/godotengine/godot/commit/b9008f3d517c3ceea565a3467064a15fdd91efca).
+
+## Highlights of 4.2.2 RC 1
+
+**75 contributors** submitted around **140 improvements** for this release. You can review the complete list of changes with our [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#4.2.2-rc1), which contains links to relevant commits and PRs for this and every previous release.
+
+Naturally, Godot 4.2.2 includes all improvements available in the 4.1.4 release. On top of that, a number of its own specific issues has been addressed. We also included more significant documentation changes, including an overhaul of `Node` ([GH-68560](https://github.com/godotengine/godot/pull/68560)) and `AABB` documentation ([GH-87114](https://github.com/godotengine/godot/pull/87114)).
+
+- If you have been experiencing issues opening large tilemaps due to preview generation, this release addresses those issues ([GH-87470](https://github.com/godotengine/godot/pull/87470)).
+
+- Several visualizations in the 3D editor have been improved, including origin lines ([GH-83895](https://github.com/godotengine/godot/pull/83895)) and `Curve3D`-related debug information ([GH-83698](https://github.com/godotengine/godot/pull/83698)). Origin lines should appear thicker and without aliasing now!
+
+- Multiple animation issues have been addressed following the changes introduced in Godot 4.2 with the `AnimationMixer` node ([GH-86046](https://github.com/godotengine/godot/pull/86046), [GH-86221](https://github.com/godotengine/godot/pull/86221), [GH-86227](https://github.com/godotengine/godot/pull/86227), [GH-86718](https://github.com/godotengine/godot/pull/86718), and others).
+
+- A new `PackedRealArray` type is now available for module developers who also target GDExtension ([GH-86324](https://github.com/godotengine/godot/pull/86324)).
```

Comment:
```suggestion
- A new `PackedRealArray` type alias is now available for module developers who also target GDExtension ([GH-86324](https://github.com/godotengine/godot/pull/86324)).
```
Makes it clearer we're not adding a whole new Variant type of bloat just for this.